framework/src/main/java/org/checkerframework/common/value/ValueAnnotatedTypeFactory.java

```diff
@@ -2327,4 +2232,37 @@ public AnnotatedTypeMirror getDummyAssignedTo(ExpressionTree expressionTree) {
         }
         return null;
     }
+
+    /**
+     * Evaluates a given annotation to its primitive integral TypeKind equivalent.
+     *
+     * @param type {@link javax.lang.model.type.TypeKind}
```

Comment:
Instead of repeating the type, maybe `the type to convert`.

Reply:
IMHO, if the javadoc is supposed to be helpful linking to the class from which the constants are used is more helpful than just saying "the  type to convert"

Reply:
The Javadoc automatically links to the type of each parameter, so there is no need to add a second link adjacent to it.  I agree with @wmdietl that "the type to convert" is better.

(If Javadoc didn't already link to the type, I would agree that such a link is useful.)